shared-config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "breakpoints": [
+    {
+      "name": "bkpt-site--x-small",
+      "threshold": 320
```

Comment:
Format feels a bit fragile.

Could it be simplified to key/values that can be namespaced?

Something like:

```yaml
breakpoints:
  site:
    x-small: 320
    small: 480
```

which in SCSS could become `breakpoints.site.x-small` and `breakpoints.site.small`.

(Could even 'template' the SCSS/JS etc (something like `$bkpt-site--x-small: {{ breakpoints.site.x-small }}`), so the resulting value doesn't have to be committed.)

Reply:
Interesting idea, will have a play.